.github/workflows/maven-release-perform.yml

```diff
@@ -16,22 +16,28 @@ concurrency:
 
 jobs:
   perform:
-    name: Release perform {{ inputs.version }}
+    name: Release perform ${{ inputs.version }}
     runs-on: ubuntu-latest
     steps:
       - name: Checkout the latest code
         uses: actions/checkout@v3
         with:
           fetch-depth: 0
+          ref: master
       - name: Setup local maven cache
         uses: actions/cache@v2
         with:
           path: ~/.m2/repository
           key: checkstyle-maven-cache-${{ hashFiles('**/pom.xml') }}
-      - name: Set up JDK 11
-        uses: actions/setup-java@v3
-        with:
-          java-version: 11
+      - name: Configure GPG
+        env:
+          GPG_SECRET_KEY: ${{ secrets.GPG_SECRET_KEY }}
```

Comment:
You will have to export your secret key like this:
```sh
gpg --list-secret-keys
gpg --output private.pgp --armor --export-secret-key <email-here>
cat private.pgp
```
Then copy whole file contents(including `-----BEGIN PGP PRIVATE KEY BLOCK-----` and `-----END PGP PRIVATE KEY BLOCK-----`) and create new actions secret `GPG_SECRET_KEY`.

From: https://unix.stackexchange.com/a/482559